AirCasting/Utils/LocationStates.swift

```diff
@@ -0,0 +1,9 @@
+// Created by Lunar on 14/07/2021.
+//
+
+import Foundation
+
+enum LocationSates {
```

Comment:
typo "Sates" -> "States". Also plural form is not really a thing with enums. `LocationState` would be perfect